packages/tools/devtools/devtools-browser-extension/e2e-tests/app/app.tsx

```diff
@@ -12,81 +12,76 @@ import {
 	IDevtools,
 	initializeDevtools,
 } from "@fluid-experimental/devtools";
-import { IFluidContainer, ContainerSchema } from "@fluidframework/fluid-static";
-import { SharedMap } from "@fluidframework/map";
-import { TinyliciousClient } from "@fluidframework/tinylicious-client";
-import { SharedString } from "@fluidframework/sequence";
+import { FluidContainer, IFluidContainer, RootDataObject } from "@fluidframework/fluid-static";
+//import { SharedMap } from "@fluidframework/map"
+import { SessionStorageModelLoader, StaticCodeLoader } from "@fluid-example/example-utils";
+import { CollaborativeTextContainerRuntimeFactory, ICollaborativeTextAppModel } from "./container";
 
-// Define the schema of our Container.
-// This includes the DataObjects we support and any initial DataObjects we want created
-// when the container is first created.
-const containerSchema: ContainerSchema = {
-	initialObjects: {
-		rootMap: SharedMap,
-	},
-	dynamicObjectTypes: [SharedMap, SharedString],
-};
 // Initialize the Devtools logger
 const logger = new DevtoolsLogger();
 
-// Initialize the Tinylicious client
-const client = new TinyliciousClient({ logger });
-
 // Initialize Devtools
 const devtools = initializeDevtools({ logger: logger });
 
-// Initialize the CollaborativeText
-const text = await getCollaborativeText(client);
-
-// Register the container with Devtools
-getContainerInfo(client).then((containerInfo) => {
-	const container = containerInfo.container;
-	registerContainerWithDevtools(devtools, container, "e2e-test-container");
-	const rootMap = container.initialObjects.rootMap as SharedMap;
-	rootMap.set("shared-text", text.handle);
+// Render the text area in the DOM
+createContainerAndRenderInElement().then(({ fluidContainer, model }) => {
+	// Register the container with Devtools
+	registerContainerWithDevtools(devtools, fluidContainer, "e2e-test-container");
 });
 
-ReactDOM.render(
-	<React.StrictMode>
-		<div className="text-area">
-			<CollaborativeTextArea sharedStringHelper={new SharedStringHelper(text)} />
-		</div>
-	</React.StrictMode>,
-	document.querySelector("#content"),
-	() => {
-		console.log("App rendered!");
-	},
-);
+/**
+ * This is a helper function for loading the page. It's required because getting the Fluid Container
+ * requires making async calls.
+ */
+async function createContainerAndRenderInElement() {
```

Comment:
This is missing a return type. I'm surprised the linter didn't complain about this. We should add the return type here.

Reply:
Added